### PR TITLE
fix(ui): Remove inset shadows

### PR DESCRIPTION
**Pull request by vuluongj20**
Removing inset shadows, which mostly appear in input fields across the app. See comments below for before-after screenshots.

### CODE REVIEW

File: static/app/components/checkboxFancy/checkboxFancy.tsx [hunk 1]
```diff
@@ -46,7 +46,6 @@ const CheckboxFancy = styled(
   display: flex;
   align-items: center;
   justify-content: center;
-  box-shadow: 1px 1px 5px 0px rgba(0, 0, 0, 0.05) inset;
```
Comment (vuluongj20) [inline static/app/components/checkboxFancy/checkboxFancy.tsx:49]: Screen Shot 2021-09-28 at 3 59 48 PM

File: static/app/components/errorRobot.tsx [hunk 2]
```diff
@@ -146,7 +146,6 @@ const ErrorRobotWrapper = styled('div')<{gradient: boolean}>`
   display: flex;
   justify-content: center;
   font-size: ${p => p.theme.fontSizeExtraLarge};
-  box-shadow: inset 0 1px 3px rgba(0, 0, 0, 0.08);
```
Comment (vuluongj20) [inline static/app/components/errorRobot.tsx:149]: Screen Shot 2021-09-28 at 4 00 21 PM

(see: the slight inset shadows around the outer borders)

File: static/app/components/forms/selectControl.tsx [hunk 3]
```diff
@@ -137,7 +137,6 @@ function SelectControl<OptionType extends GeneralSelectValue = GeneralSelectValu
         color: theme.formText,
         background: theme.background,
         border: `1px solid ${theme.border}`,
-        boxShadow: `inset ${theme.dropShadowLight}`,
```
Comment (vuluongj20) [inline static/app/components/forms/selectControl.tsx:140]: Screen Shot 2021-09-28 at 4 01 45 PM

File: static/app/components/modals/dashboardWidgetQuerySelectorModal.tsx [hunk 4]
```diff
@@ -107,7 +107,6 @@ const OpenInDiscoverButton = styled(Button)`
 
 const Container = styled('div')`
   border: 1px solid ${p => p.theme.border};
-  box-shadow: inset ${p => p.theme.dropShadowLight};
```
Comment (vuluongj20) [inline static/app/components/modals/dashboardWidgetQuerySelectorModal.tsx:110]: Screen Shot 2021-09-28 at 4 02 11 PM

(see: the input fields)

File: static/app/components/progressBar.tsx [hunk 5]
```diff
@@ -28,7 +28,6 @@ const getVariantStyle = ({
       height: 24px;
       border-radius: 24px;
       border: 1px solid ${theme.border};
-      box-shadow: inset 0px 1px 3px rgba(0, 0, 0, 0.06);
```
Comment (vuluongj20) [inline static/app/components/progressBar.tsx:31]: Screen Shot 2021-09-28 at 4 03 03 PM

File: static/app/components/radio.tsx [hunk 6]
```diff
@@ -30,7 +30,6 @@ const Radio = styled('input')<Props>`
   align-items: center;
   justify-content: center;
   border: 1px solid ${p => p.theme.border};
-  box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.04);
```
Comment (vuluongj20) [inline static/app/components/radio.tsx:33]: Screen Shot 2021-09-28 at 4 03 24 PM

File: static/app/components/searchBar.tsx [hunk 7]
```diff
@@ -126,7 +126,6 @@ class SearchBar extends React.PureComponent<Props, State> {
 const StyledInput = styled(Input)`
   width: ${p => (p.width ? p.width : undefined)};
   &.focus-visible {
-    box-shadow: inset 0 2px 0 rgba(0, 0, 0, 0.04), 0 0 6px rgba(177, 171, 225, 0.3);
```
Comment (vuluongj20) [inline static/app/components/searchBar.tsx:129]: Screen Shot 2021-09-28 at 4 03 42 PM

(on `:focus-visible`)

File: static/app/components/switchButton.tsx [hunk 8]
```diff
@@ -67,7 +67,6 @@ const SwitchButton = styled('button')<StyleProps>`
   padding: 0;
   border: 1px solid ${p => p.theme.border};
   position: relative;
-  box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.04);
```
Comment (vuluongj20) [inline static/app/components/switchButton.tsx:70]: Screen Shot 2021-09-28 at 4 04 29 PM

File: static/app/styles/input.tsx [hunk 9]
```diff
@@ -19,7 +19,6 @@ const inputStyles = (props: Props) =>
     background: ${props.theme.background};
     border: 1px solid ${props.theme.border};
     border-radius: ${props.theme.borderRadius};
-    box-shadow: inset ${props.theme.dropShadowLight};
```
Comment (vuluongj20) [inline static/app/styles/input.tsx:22]: Screen Shot 2021-09-28 at 4 04 55 PM

File: static/app/views/eventsV2/table/arithmeticInput.tsx [hunk 10]
```diff
@@ -278,7 +278,6 @@ export default class ArithmeticInput extends PureComponent<Props, State> {
 
 const Container = styled('div')<{isOpen: boolean}>`
   border: 1px solid ${p => p.theme.border};
-  box-shadow: inset ${p => p.theme.dropShadowLight};
```
Comment (vuluongj20) [inline static/app/views/eventsV2/table/arithmeticInput.tsx:281]: Screen Shot 2021-09-28 at 4 05 20 PM

File: static/app/views/settings/components/settingsSearch/index.tsx [hunk 11]
```diff
@@ -75,8 +75,6 @@ const SearchInput = styled('input')`
   border-radius: 30px;
   height: 28px;
 
-  box-shadow: inset ${p => p.theme.dropShadowLight};
```
Comment (vuluongj20) [inline static/app/views/settings/components/settingsSearch/index.tsx:78]: Screen Shot 2021-09-28 at 4 06 25 PM

File: static/app/components/modals/dashboardWidgetQuerySelectorModal.tsx (hunk 4 above)
Comment (edwardgou-sentry) [reply static/app/components/modals/dashboardWidgetQuerySelectorModal.tsx:110]: 👍

**Review by edwardgou-sentry — APPROVED** [review]
`dashboardWidgetQuerySelectorModal.tsx` change looks good to me. Don't think I see any issues with the other changes but I'll let someone from core ui to confirm

**Pull request closed without merging**